src/sagemaker_inference/default_inference_handler.py

```diff
@@ -85,4 +85,7 @@ def default_output_fn(self, prediction, accept):  # pylint: disable=no-self-use
             obj: prediction data.
 
         """
-        return encoder.encode(prediction, accept), accept
+        for content_type in utils.parse_accept(accept):
+            if content_type in encoder.SUPPORTED_CONTENT_TYPES:
+                return encoder.encode(prediction, content_type), content_type
+        return encoder.encode(prediction, content_types.JSON), content_types.JSON
```

Comment:
I've implemented this in `default_inference_handler`, but I could've just as easily put it in `transformer`.

Reply:
Also, could use recommendations on how to more robustly test this behavior

Reply:
> I've implemented this in `default_inference_handler`, but I could've just as easily put it in `transformer`.

I think it makes more sense here - that way, users can still override it if they want to for some reason